### PR TITLE
Switches dev env to qpid

### PR DESCRIPTION
**Pull request by bmbouter**
Adds a new role called kombu_fixup which provides a source install of
qpid-tools and kombu.

closes #2259

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -94,3 +94,6 @@ Vagrantfile*
 .vagrant
 .dnf-cache
 ansible/*.retry
+
+# PyCharm
```
Comment (daviddavis) [inline .gitignore]: vim > pycharm 😄

File: Vagrantfile.example [hunk 2]
```diff
@@ -39,6 +39,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision "ansible" do |ansible|
       # ansible.verbose = "vvv"
       ansible.playbook = "ansible/pulp-from-source.yml"
+      # ansible.start_at_task = "ansible task name here"
+      # ansible.extra_vars = { pulp_venv: "/home/vagrant/.virtualenvs/pulp" }
```
Comment (daviddavis) [inline Vagrantfile.example]: Did you mean to commit these commented out lines?

**Comment by daviddavis** [comment]
Would it be easy to add like a setting (perhaps in Vagrantfile) where a user could specify which message transport he/she wants to use (default would be qpid)?

File: Vagrantfile.example (hunk 2 above)
Comment (bmbouter) [reply Vagrantfile.example]: I did, I think they will be useful settings again. Really L#42 is the useful one, but then you also need L#43.

**Comment by daviddavis** [comment]
Looks good 👍

**Comment by bmbouter** [comment]
Yes I can add a setting that will allow the user to select qpid or rabbitmq and then default to qpid.

**Comment by bmbouter** [comment]
@daviddavis I added the commented variable to Vagrantfile.example. If you uncomment that it will force it to use rabbitmq.

@dkliban I also switched it to F27.

I'm doing my final testing of both broker types now.

**Comment by bmbouter** [comment]
OK I tested both environments. Both rabbitmq and qpid environments worked with F27 with the pulp_file sync commands. It's ready for independent testing and final review.

**Comment by bmbouter** [comment]
Thanks for the testing! I'm also sending an email to pulp-dev explaining the change and variable to revert to rabbitmq.

**Comment by daviddavis** [comment]
Awesome 👍

**Pull request merged**